docs/ensembl-help/using-ensembl/getting-ensembl-data/tools-and-apis/refget/refget.md

```diff
@@ -0,0 +1,246 @@
3448.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05090000248.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05095018813.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05100071832.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05105031936.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05110063010.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05115001460.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05120029648.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05125100593.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05130014039.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05135008180.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05140011277.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05145037430.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05150000717.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05155102176.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05160002764.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05165027060.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05170002082.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05175053786.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05180005414.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05185019879.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05190002160.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05195002070.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05200054588.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05205003605.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05210086440.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05215035545.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05220081924.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05225045612.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05230040062.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05235062312.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05240024690.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05245059294.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05250001624.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05255015884.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05260098332.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05265094935.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05270027733.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05275001551.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05280005213.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05300104059.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05305089630.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05310013165.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05315008972.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05520061176.1", "naming_authority": "Ensembl"}, {"alias": "ENSP05525016640.1", "naming_authority": "Ensembl"}], "length": 1218, "md5": "84706f0ee2a4d23ae4050036e62dac5d", "trunc512": "d29ca0d14bb727c2c3593126ff5e2c510ec3097f2b2bc9e5"}}
```

Comment:
This is quite verbose.  Could it be trimmed to give one alias example?

Reply:
Done